esp/esp/program/modules/handlers/jsondatamodule.py

```diff
@@ -781,14 +794,18 @@ def stats(prog):
         reg_hours = JSONDataModule.calc_hours(reg_classes)
         app_classes = prog.classes().filter(status__gt=0, sections__status__gt=0).exclude(category__category='Lunch').annotate(num_sections=Count('sections'), subject_duration=Sum('sections__duration'), subject_students=Sum('sections__enrolled_students')).values('num_sections', 'subject_duration', 'subject_students', 'class_size_max')
         app_hours = JSONDataModule.calc_hours(app_classes)
+        sched_sections = prog.sections().filter(status__gt=0, meeting_times__isnull=False).exclude(parent_class__category__category='Lunch').values('duration', 'enrolled_students', 'id')
+        sched_hours = JSONDataModule.calc_section_hours(sched_sections)
         vitals["hournum"] = []
         vitals["hournum"].append(("Total # of Class-Hours (registered)", reg_hours["class-hours"]))
         vitals["hournum"].append(("Total # of Class-Hours (approved)", app_hours["class-hours"]))
+        vitals["hournum"].append(("Total # of Class-Hours (scheduled)", sched_hours["class-hours"]))
         vitals["hournum"].append(("Total # of Class-Student-Hours (registered)", reg_hours["class-student-hours"]))
         vitals["hournum"].append(("Total # of Class-Student-Hours (approved)", app_hours["class-student-hours"]))
+        vitals["hournum"].append(("Total # of Class-Student-Hours (scheduled)", sched_hours["class-student-hours"]))
         vitals["hournum"].append(("Total # of Class-Student-Hours (enrolled)", reg_hours["class-registered-hours"]))
         if app_hours["class-student-hours"]:
```

Comment:
Does this need to be sched_hours instead of app_hours now?

Reply:
Yes, thanks!